src/base/utils/string.cpp

```diff
@@ -181,6 +184,12 @@ QString Utils::String::fromDouble(const double n, const int precision)
     return QLocale::system().toString(std::floor(n * prec) / prec, 'f', precision);
 }
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+QString Utils::String::wildcardToRegex(const QString &pattern)
```

Comment:
It seems this function name could be improved to be more precise, I would rename it to `wildcardToRegexPattern`.
The current name gives me the impression of returning a `QRegularExpression` variable.

Reply:
This function should be renamed as well, otherwise it will fail in Qt6.

Reply:
> This function should be renamed as well, otherwise it will fail in Qt6.

Fixed.